src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -156,16 +156,17 @@ extension ExposureSubmissionCoordinator {
 			if shouldDismiss { self?.navigationController?.dismiss(animated: true) }
 		}
 	}
-
-	func showOverviewScreen() {
-		let vc = ExposureSubmissionOverviewViewController(
+	
+	// New submission flow:
+	func getFetchTestResultViewController() -> ExposureSubmissionFetchTestResultViewController {
+		let viewModel = ExposureSubmissionFetchTestResultViewModel(
 			onQRCodeButtonTap: { [weak self] in self?.showQRInfoScreen() },
 			onTANButtonTap: { [weak self] in self?.showTanScreen() },
 			onHotlineButtonTap: { [weak self] in self?.showHotlineScreen() }
 		)
-		push(vc)
+		return ExposureSubmissionFetchTestResultViewController(viewModel)
 	}
-
+	
```

Comment:
As quoted in the comment, this looks much more like a new flow from here on.
We should discuss in the team whether the logic at this point should be transferred to a new flow. The murderous view is a good starting point for this.